bitbots_bringup/launch/remote_vision.launch

```diff
@@ -0,0 +1,23 @@
+<launch>
+  <!-- Launch the vision on a remote device-->
+  <arg name="sim" default="false" doc="true: activates simulation time, switches to simulation color settings and deactivates launching of an image provider" />
+  <arg name="camera" default="true" doc="true: launches an image provider to get images from a camera (unless sim:=true)" />
+  <arg name="basler" default="true" doc="true: launches the basler camera driver instead of the wolves image provider" />
+  <arg name="dummyball" default="false" doc="true: does not start the ball detection to save resources" />
+  <arg name="debug" default="false" doc="true: activates publishing of several debug images" />
+  <arg name="use_game_settings" default="false" doc="true: loads additional game settings" />
+
+  <node pkg="bitbots_bringup" type="launch_on_remote.sh"
+    args="jetson
```

Comment:
This file is named _remote_vision.launch_ and the description says that vision will be launched on _a remote device_ but you're always connecting to _jeston_ which is a hostname only available when run on a robot computer.

Please add a launch argument for the target so the user can execute this launch file on his machine as well. _jetson_ would be a sensible default value for said argument though.